fvtest/compilertest/tests/X86OpCodesTest.cpp

```diff
@@ -4440,6 +4440,10 @@ X86OpCodesTest::UnsupportedOpCodesTests()
    addUnsupportedOpCodeTest(_numberOfUnaryArgs, TR::su2a, "su2a", _argTypesUnaryShort, TR::Address);
    addUnsupportedOpCodeTest(_numberOfUnaryArgs, TR::lu2a, "lu2a", _argTypesUnaryLong, TR::Address);
 
+   //bmul, smul
+   addUnsupportedOpCodeTest(_numberOfBinaryArgs, TR::bmul, "bMul", _argTypesBinaryByte, TR::Int8);
+   addUnsupportedOpCodeTest(_numberOfBinaryArgs, TR::smul, "sMul", _argTypesBinaryByte, TR::Int16);
+
```

Comment:
Are these really needed? These kinds of tests are going to be removed in the not too distant future.